Jenkinsfile

```diff
@@ -1,3 +1 @@
-Map pipelineParams = [ "projectType" : "Extensions" ]
-
-runtimeBuild(pipelineParams)
\ No newline at end of file
+onpremBuild()
```

Comment:
this jenkins file is for the connectivity pipeline. Not the runtime pipeline

Reply:
We worked a lot in this PR in order to it were able be run in this pipeline with Java 17. In this case for this file @aiannucci was tuning until it worked. Maybe if you have the correct configuration for this Jenkinsfile you can give us and try to it works properly.

Reply:
Oh I confused conenctivity pipeline and runtime pipeline configs